recipes/uwsgi/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: uwsgi
+  version: 2.0.12
+
+source:
+  fn: "uwsgi-2.0.12.tar.gz"
+  url: "https://pypi.python.org/packages/0c/1c/44849e293e367a157f1ad863cee02b4b865840543254d8fae3ecdebdbdb9/uwsgi-2.0.12.tar.gz"
+  md5: 1451cab954bad0d7d7429e4d2c84b5df
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - python
+    # - jansson
+    - libxml2
+    - openssl 1.0.2*
+    - pcre
+    - yaml
+    - zlib
+  run:
+    - python
+    # - jansson
+    - libxml2
+    - openssl 1.0.2*
+    - pcre
+    - yaml
+    - zlib
+
+test:
+  commands:
+    - uwsgi --version
+
+about:
+  home: https://github.com/unbit/uwsgi
```

Comment:
I thought supporting `dev_url` and `doc_url` were forthcoming, but looks like it was long ago done.
https://github.com/conda/conda-build/pull/831

Would you prefer those keys here?